doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -12,31 +12,34 @@ The ``Scheme`` command is a high-level tool for generating automatically
 (possibly mutual) induction principles for given types and sorts. Its
 syntax follows the schema:
 
-.. cmd:: Scheme @ident__1 := Induction for @ident__2 Sort @sort {* with @ident__i := Induction for @ident__j Sort @sort}
+.. cmd:: Scheme {? @ident__def := } @scheme_kind {* with {? @ident__def := } @scheme_kind }
```

Comment:
Since there is no longer any `ident` besides the `@ident__def`, I don't think keeping the `__def` is that useful. I would remove it.